docker/docker-bake.hcl

```diff
@@ -92,13 +92,19 @@ target "front-devel" {
   inherits = ["base", "base-front-devel"]
   context = "front"
   dockerfile = "docker/Dockerfile.devel"
+  contexts = {
+    test_data = "./tests/data"
+  }
 }
 
 target "base-front-nginx" {}
 target "front-nginx" {
   inherits = ["base", "base-front-nginx"]
   context = "front"
   dockerfile = "docker/Dockerfile.nginx"
+  contexts = {
+    test_data = "./tests/data"
```

Comment:
I'm not sure why we need it on both nginx and dev, if it's required on front-nginx, maybe we should add a front-nginx-test with a custom target on the HCL file.

Reply:
done!